packages/components/package.json

```diff
@@ -12,12 +11,21 @@
     "bugs": {
         "url": "https://github.com/frontify/fondue/issues"
     },
-    "main": "src/index.ts",
-    "types": "src/index.ts",
-    "engines": {
```

Comment:
engine and package manager are only needed at the root of the project `package.json`

Reply:
We actually need it when installing the package alone (without `@frontify/fondue`)

Reply:
Not if you install directly `@frontify/fondue-ui` (which is also a possibility)